package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@centrifuge/tinlake-js",
+  "name": "@centrifuge/tinlake-js-v3",
```

Comment:
Why this change?

Reply:
Ah I need to revert it, that's needed for making `npm link` work locally..